core/option/options.go

```diff
@@ -620,17 +621,18 @@ func (opt OptMaxTime) Option(d *bson.Document) error {
 	return nil
 }
 
-func (OptMaxTime) aggregateOption()         {}
-func (OptMaxTime) countOption()             {}
-func (OptMaxTime) distinctOption()          {}
-func (OptMaxTime) findOption()              {}
-func (OptMaxTime) findOneOption()           {}
-func (OptMaxTime) findOneAndDeleteOption()  {}
-func (OptMaxTime) findOneAndReplaceOption() {}
-func (OptMaxTime) findOneAndUpdateOption()  {}
-func (OptMaxTime) listIndexesOption()       {}
-func (OptMaxTime) dropIndexesOption()       {}
-func (OptMaxTime) createIndexesOption()     {}
+func (OptMaxTime) aggregateOption()              {}
+func (OptMaxTime) countOption()                  {}
+func (OptMaxTime) distinctOption()               {}
+func (OptMaxTime) findOption()                   {}
+func (OptMaxTime) findOneOption()                {}
+func (OptMaxTime) findOneAndDeleteOption()       {}
+func (OptMaxTime) findOneAndReplaceOption()      {}
+func (OptMaxTime) findOneAndUpdateOption()       {}
+func (OptMaxTime) listIndexesOption()            {}
+func (OptMaxTime) dropIndexesOption()            {}
+func (OptMaxTime) createIndexesOption()          {}
+func (OptMaxTime) estimatedDocumentCountOption() {}
```

Comment:
We can remove this line since we no longer have `EstimatedDocumentCountOptioner`.

Reply:
Done.